messages/php/src/Id/FallbackUuidGenerator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cucumber\Messages\Id;
+
+final class FallbackUuidGenerator implements IdGenerator
+{
+    /**
+     * Inspired by
+     * @see https://github.com/clue/reactphp-clickhouse
+     */
+    public function newId(): string
+    {
+        $bytes = random_bytes(16);
```

Comment:
What is the quality of this random source?

Reply:
According to [PHP manual](https://www.php.net/manual/en/function.random-bytes.php): 
"Generates an arbitrary length string of cryptographic random bytes that are suitable for cryptographic use, such as when generating salts, keys or initialization vectors."

Reply:
It's slightly platform dependent bit is meant to be "for reading CS random bytes from the system CSPRNG."

The original RFC is here https://wiki.php.net/rfc/csrandombytes

Reply:
So TL;DR it could be poor quality if the system doesn't have a high quality random source, or if the person compiling the PHP runtime somehow disabled the available sources.

Reply:
Since Cucumber is a testing framework, I think we're okay then. At worst the messages output will be unusable then.